tox.ini

```diff
@@ -6,7 +6,7 @@ envlist =
        {py27,py32,py33,py34}-django{17,18,master}
 
 [testenv]
-commands = ./runtests.py --fast
+commands = ./runtests.py --fast {posargs}
```

Comment:
Allows positional arguments to be passed from running `tox <...>` on the command line. It's a valid change. Not properly part of this PR, but fine all the same. :smile: